client/admin-pointers.js

```diff
@@ -22,17 +20,29 @@ jQuery( document ).ready( function( $ ) {
 			return;
 		}
 
+		const target = $( pointer.target );
+
 		const options = $.extend( pointer.options, {
 			close: function() {
 				$.post( ajaxurl, {
 					pointer: pointer.id,
 					action: 'dismiss-wp-pointer',
 				} );
+
+				if ( pointer.dim ) {
+					$( '#wcs-pointer-page-dimmer' ).fadeOut( 500 );
```

Comment:
In the effort of getting this landed sooner than later this can likely be done later, but..

Should this check to see if the next pointer is dimmed as well, and skip the fade out if so?